.github/workflows/golangci-test.yml

```
@@ -0,0 +1,32 @@
+name: golangci-test
```

Rename? The other action is named after the specific tool, `golangci-lint`.
```suggestion
name: go-test
```
Rename the file too.

---

Renamed.